src/wallet/rpcwallet.cpp

```diff
@@ -583,6 +583,57 @@ static UniValue signmessage(const JSONRPCRequest& request)
     return EncodeBase64(vchSig.data(), vchSig.size());
 }
 
+static CAmount GetReceived(interfaces::Chain::Lock& locked_chain, CWallet * const wallet, const UniValue& params, bool by_label) EXCLUSIVE_LOCKS_REQUIRED(wallet->cs_wallet)
```

Comment:
Variable naming: s/`wallet`/`pwallet` (see e.g. `ListReceived()` and `SendMoney()` in the same file) to be more consistent